controllers/configuration_controller.go

```diff
@@ -205,20 +205,23 @@ type TFConfigurationMeta struct {
 	Credentials           map[string]string
 
 	// TerraformImage is the Terraform image which can run `terraform init/plan/apply`
-	TerraformImage            string
+	TerraformImage string
+	// TerraformContainerName is the name of the container that executes the terraform in the pod
+	TerraformContainerName    string
 	TerraformBackendNamespace string
 	BusyboxImage              string
 	GitImage                  string
 }
 
 func initTFConfigurationMeta(req ctrl.Request, configuration v1beta1.Configuration) *TFConfigurationMeta {
 	var meta = &TFConfigurationMeta{
-		Namespace:           req.Namespace,
-		Name:                req.Name,
-		ConfigurationCMName: fmt.Sprintf(TFInputConfigMapName, req.Name),
-		VariableSecretName:  fmt.Sprintf(TFVariableSecret, req.Name),
-		ApplyJobName:        req.Name + "-" + string(TerraformApply),
-		DestroyJobName:      req.Name + "-" + string(TerraformDestroy),
+		Namespace:              req.Namespace,
+		Name:                   req.Name,
+		ConfigurationCMName:    fmt.Sprintf(TFInputConfigMapName, req.Name),
+		VariableSecretName:     fmt.Sprintf(TFVariableSecret, req.Name),
+		ApplyJobName:           req.Name + "-" + string(TerraformApply),
+		DestroyJobName:         req.Name + "-" + string(TerraformDestroy),
+		TerraformContainerName: "terraform-executor",
```

Comment:
As `TerraformContainerName` works as a constant, how about we set it as a const instead of a filed of `TFConfigurationMeta`?

Reply:
Yes, actually thats how i started, but then I was conflicted as where to declare it. I was going to declare it closer to the logging.go, but I also realized that we have reference of that string in the configuration-controller.go where we use the same string for creating the job. 
Would you prefer the constant declared in the logging.go and referred in the configuration-controller? The other way would cause cyclic dependency in packages

Reply:
Or did you mean just declare it as constant in the configuration-controller.go and pass the variable with the method signature change?

Reply:
@zzxwill can you check now